packages/core/admin/admin/src/pages/UsecasePage/index.js

```diff
@@ -0,0 +1,148 @@
+import React, { useState, useEffect } from 'react';
+import axios from 'axios';
+import { useIntl } from 'react-intl';
+import { useHistory } from 'react-router-dom';
+import styled from 'styled-components';
+import { pxToRem, useNotification } from '@strapi/helper-plugin';
+import { Main } from '@strapi/design-system/Main';
+import { Flex } from '@strapi/design-system/Flex';
+import { Box } from '@strapi/design-system/Box';
+import { Stack } from '@strapi/design-system/Stack';
+import { Typography } from '@strapi/design-system/Typography';
+import { Select, Option } from '@strapi/design-system/Select';
+import { TextInput } from '@strapi/design-system/TextInput';
+import { TextButton } from '@strapi/design-system/TextButton';
+import { Button } from '@strapi/design-system/Button';
+import Logo from '../../components/UnauthenticatedLogo';
+import UnauthenticatedLayout, { LayoutContent } from '../../layouts/UnauthenticatedLayout';
+
+export const options = [
+  { label: 'Front-end developer', value: 'front_end_developer' },
+  { label: 'Back-end developer', value: 'back_end_developer' },
+  { label: 'Full-stack developer', value: 'full_stack_developer' },
+  { label: 'Content Manager', value: 'content_manager' },
+  { label: 'Content Creator', value: 'content_creator' },
+  { label: 'Other', value: 'other' },
+];
+
+const TypographyCenter = styled(Typography)`
+  text-align: center;
+`;
+
+const UsecasePage = () => {
+  const toggleNotification = useNotification();
+  const { push, location } = useHistory();
+  const { formatMessage } = useIntl();
+  const [persona, setPersona] = useState();
+  const [personaOther, setPersonaOther] = useState('');
+
+  const isOther = persona === 'other';
+  const isComingFromRegister = location.state?.fromRegister;
+
+  useEffect(() => {
+    if (!isComingFromRegister) {
+      push('/');
+    }
+  }, [isComingFromRegister, push]);
+
+  const handleSubmit = skipPersona => {
+    try {
+      const {
+        state: { email, firstAdmin, firstname },
+      } = location;
+
+      axios({
+        method: 'POST',
+        url: 'https://analytics.strapi.io/register',
+        data: {
+          email,
+          username: firstname,
+          firstAdmin,
+          persona: skipPersona ? undefined : persona,
+          personaOther: skipPersona ? undefined : personaOther,
+        },
+      });
+
+      toggleNotification({
+        type: 'success',
+        message: {
+          id: 'Usecase.notification.success.project-created',
+          defaultMessage: 'Project has been successfully created',
+        },
+      });
+      push('/');
+    } catch (err) {
+      // Silent
+    }
+  };
+
+  if (!isComingFromRegister) {
+    return null;
+  }
+
+  return (
+    <UnauthenticatedLayout>
+      <Main>
+        <LayoutContent>
+          <form onSubmit={() => handleSubmit(false)}>
```

Comment:
For accessibility reasons it would be quite nice, if users could submit the form (by hitting enter e.g.). Could you move `setPersona` to be handled once the form is submitted?

Reply:
Something is wrong with the Skip to content button in this page. Focusing "SkipToContent" button and pressing "Enter" submits the form instead of focusing the main content of the page. 

![Screenshot 2022-02-15 at 11 17 45](https://user-images.githubusercontent.com/7756284/154041561-ebf600fd-049f-468a-8c56-4fdcbaf698df.png)

Reply:
Edit: 
Select clear button type submit caused this issue
Commented clearLabel and onClear for now until 1.0.0 DS releases